### PR TITLE
Store token in session instead of cookies, just stored session id in cookies

### PR DESCRIPTION
**Pull request by harshkothari410**
- Token is stored in Session
- session id is stored in cookies
- user info in session as well
- removed all occurrences of getCookie authorization 
- fixed test

### CODE REVIEW

File: utils/jwtUtil.js [hunk 1]
```diff
@@ -89,12 +89,7 @@ function checkTokenRecord(t) {
  * @param  {Function} cb - callback function
  */
 function verifyToken(req, cb) {
-  const authorization = req.headers.authorization;
-  let token;
-
-  if (authorization) {
-    token = req.headers.authorization;
-  }
+  const token = req.session.token || req.headers.authorization;
```
Comment (iamigo) [inline utils/jwtUtil.js]: Yes!

**Comment by harshkothari410** [comment]
I created Review App @iamigo : http://refocus-pr-305.herokuapp.com/

**Comment by harshkothari410** [comment]
@iamigo without SSO I successfully run it. You can also check it out. With SSO I do not know how to run.

**Comment by iamigo** [comment]
FYI @harshkothari410 docs on setting up SSO (but you don't need to do it in your review app): https://salesforce.github.io/refocus/docs/80-SSOSetup.html

File: view/loadView.js [hunk 2]
```diff
@@ -306,9 +306,9 @@ module.exports = function loadView(app, passport) {
 
         // return token
         const token = jwtUtil.createToken(req.user.name, req.user.name);
+        req.session.token = token;
```
Comment (pallavi2209) [inline view/loadView.js]: So now if we don't need to send plain token back, can we remove /authenticate route and make use of /v1/authenticate instead?

**Comment by pallavi2209** [comment]
We are storing session in cookie for SSO flow as well. Please check /sso/saml route in loadView. We will have to change that as well.

File: view/loadView.js (hunk 2 above)
Comment (harshkothari410) [inline view/loadView.js]: I guess we can do this without any problem. Can I try to remove that ? @pallavi2209 @iamigo

Comment (pallavi2209) [inline view/loadView.js]: Fine with me.

File: view/admin/actions/index.js [hunk 3]
```diff
@@ -152,7 +151,6 @@ function requestWithOutPayload(verb, url) {
   return new Promise((resolve, reject) => {
     superagent(verb, API_URL + url)
     .set('Content-Type', 'application/json')
-    .set('Authorization', u.getCookie('Authorization'))
```
Comment (pallavi2209) [inline view/admin/actions/index.js]: Do we still need getCookie and setCookie functions?

Comment (harshkothari410) [inline view/admin/actions/index.js]: Yes I guess because its general function for cookies not just for authorization cookies.

**Pull request merged**